app/Filament/Server/Resources/UserResource.php

```diff
@@ -264,6 +261,7 @@ public static function table(Table $table): Table
                                                             })
                                                             ->bulkToggleable()
                                                             ->label('')
+                                                            ->columns(2)
```

Comment:
Why the `->columns(2)` everywhere?

Reply:
I'm fine with this. The Invite modal has the permissions in a 2 column layout already, this is just making the edit match the create 👍

Reply:
![image](https://github.com/user-attachments/assets/dc987509-5ee4-4db7-847b-b6f7ef19fe9a)
![image](https://github.com/user-attachments/assets/a1ff9b6a-e3e4-400a-b0da-ae5e29cc0626)